### DISCUSSION TITLE
Server start/stop and container names

### DISCUSSION THREAD

**Discussion by the4tress** [edward3h/minecraft-webhook#2] [Ideas]
Love this project!

Would it be possible to also listen for server start/stop messages and pass the container name? I have multiple containers running on a server and would like to be able to know which server somebody joined.

**Comment by edward3h** [comment]
Yes, that's a reasonable idea.

I'll add some kind of message template so you can customize what's in the messages.

**Reply by the4tress** [reply]
I'm feeding all of this into a Node-RED flow and pushing to Slack. The template idea would be great!

**Reply by edward3h** [reply]
I realized this morning it would be useful to have an option to just send a JSON object rather than a formatted message, so I'll do that too. Had not heard of Node-RED so I'll take a look at that.

**Reply by the4tress** [reply]
Thanks for the responsiveness! I'll get a short video of how I'm using Node-RED put together so you can see the flow. Basically its GUI programming working with JSON objects.

**Reply by edward3h** [reply]
Pretty close to releasing the next version, but it's getting late here.
I had a play with Node-RED while testing and got an interesting flow going. Thanks for letting me know about it.

**Comment by the4tress** [comment]
Sorry to pile on, but I had one more request for parsing/returning data. I recently needed to use the user's xuid to update the `permissions.json`. Would you be able to return that as well?

**Reply by edward3h** [reply]
Yes I can do that. I made a note on #7

**Comment by edward3h** [comment]
I released 0.3. Check the main project page for the updated instructions and let me know how you get on!

**Reply by the4tress** [reply]
Awesome, thank you! I'm out this weekend but will definitely give it a shot next week.

Thanks!!
